src/applications/letters/components/ErrorableTextInput.jsx

```diff
@@ -49,7 +49,7 @@ class ErrorableTextInput extends React.Component {
 
     // Calculate max characters and display '(Max. XX characters)' when max is hit.
     if (this.props.value) {
-      if (this.props.charMax === this.props.value.length) {
+      if (this.props.charMax < this.props.value.length) {
```

Comment:
It really never made sense to me why we'd show an "error" message to the user when they've reached the maximum allowed characters when the user is prevented from entering more than the allowable limit: https://github.com/department-of-veterans-affairs/vets-website/blob/e1386c8e73dbdff6d6c91faad5cb08b189b7d530/src/applications/letters/components/ErrorableTextInput.jsx#L79